roles/datagateway-download-api/handlers/topcat-handler.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: "Re-install topcat"
```

Comment:
Is this handler used anywhere? It seems very similar to the other handler

Reply:
Nope. I've removed it now.